proto/wechaty/puppet/message.proto

```diff
@@ -114,6 +112,12 @@ message MessageSendFileResponse {
   google.protobuf.StringValue id = 1;
 }
 
+message MessageSendFileStreamRequestChunk {
```

Comment:
@windmemory Please see my new design and let me know what you think.

Thank you very much.

Reply:
It's okay for me, in my opinion this is not a perfect solution either.

Pro:
- Better structured data payload, better constraint of the data format

Con:
- More hidden rules([conversationId in first chunk, fileName second chunk, then data] vs [extra data in first chunk, then data]) when making the `MessageSendFile`

I can go with this design, in fact with this design, we choose constraint over convenience.

Reply:
Merged to #96